python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -375,6 +397,14 @@ def tags_for_partition_set(partition_set, partition):
         return {PARTITION_NAME_TAG: partition.name, PARTITION_SET_TAG: partition_set.name}
 
 
+# previous name for NodeHandle was SolidHandle
+register_serdes_tuple_fallbacks({"PipelineRun": DagsterRun})
+
+
+class PipelineRun(DagsterRun):
```

Comment:
add comment here about the purpose of this?

Reply:
Do we need to worry about breaking user's code that are relying on types?

If we have something typed as `PipelineRun` and we return a `DagsterRun`, we'll get errors, right?